src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/alerting/NewEnvelopesChecker.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.services.alerting;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.EnvelopeRepository;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.util.function.Supplier;
+
+import static java.time.DayOfWeek.SATURDAY;
+import static java.time.DayOfWeek.SUNDAY;
+import static java.time.ZoneOffset.UTC;
+import static java.util.Arrays.asList;
+
+public class NewEnvelopesChecker {
+
+    private static final Logger logger = LoggerFactory.getLogger(NewEnvelopesChecker.class);
+
+    public static final Duration TIME_WINDOW = Duration.ofHours(1);
+    public static final int START_HOUR = 10;
+    public static final int END_HOUR = 18;
```

Comment:
Maybe it would be worth adding a config section? Up to you.

Reply:
not unless we want to per environment

Reply:
Leszek's idea (about config) looks good for me